homework.md

```diff
@@ -4,38 +4,36 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+white
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
-
+Hyper Text Markup Language
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
-
+CSS is the language for describing the presentation of Web pages
 ## Question 6
```

Comment:
Hi @Maziarmajd you could also say something like; it is used for styling purposes to make websites more attractive.

Reply:
Hi Seyed, yes it is the better answer. thank you for your review

Reply:
You are welcome.